t/data/authentication.conf

```diff
@@ -215,7 +215,7 @@ cache_match=1
 description=Full Access
 class=authentication
 match=any
-condition0=memberOf,equals,CN=NOC Users,DC=ldap,DC=inverse,DC=ca
+condition0=ldap:memberOf2,equals,CN=NOC Users,DC=ldap,DC=inverse,DC=ca
```

Comment:
@VakarisZ typo?

Reply:
> @VakarisZ typo?

This is not a typo. It is used for a test in `t/unittest/authentication.t`